fvm/src/trace/mod.rs

```diff
@@ -25,7 +26,10 @@ pub enum ExecutionEvent {
         method: MethodNum,
         params: Option<IpldBlock>,
         value: TokenAmount,
+        gas_limit: u64,
+        read_only: bool,
     },
     CallReturn(ExitCode, Option<IpldBlock>),
     CallError(SyscallError),
+    InvokeActor(Cid),
```

Comment:
What's `InvokeActor`, and how does it differ from `Call`?

Reply:
Its the code CID of the invoked actor after resolving its address.

Reply:
Basically:

1. Call means actor A calls actor B, but actor B may not yet exist (and/or may never exist).
2. InvokeActor means that we actually successfully invoked actor B.

Reply:
So we'll see both in the trace for most successful calls? And there's no InvokeActorEnd? 

This all sounds fine, but please consider documenting it here.